content/applications/sales/crm/optimize/utilize_activities.rst

```diff
@@ -159,67 +161,92 @@ past due activities.
 The pipeline is most effective when it is kept up-to-date and accurate to the interactions it is
 tracking.
 
+.. _crm/activity-plans:
+
 Activity plans
---------------
+==============
+
+*Activity Plans* are preconfigured sequences of activities that can be triggered with one click,
+and replicated with little to no manual setup. Activity plans can be customized for various
+workflows, with multiple deadlines and responsible users assigned in a single plan.
+
+To create a new plan, navigate to :menuselection:`CRM app --> Configuration --> Activity Plan`.
+Click :guilabel:`New` at the top-left of the page to open a blank :guilabel:`Lead Plans` form.
+
+Enter a name for the new plan in the :guilabel:`Plan Name` field. On the :guilabel:`Activities to
+Create` tab, click :guilabel:`Add a line`.
+
+First, select an :guilabel:`Activity Type` from the drop-down menu. Click :guilabel:`Search More` to
+see a complete list of available activity types, or to create a :ref:`new one
+<crm/create-new-activity-type>`.
+
+Next, in the :guilabel:`Summary` field, enter any details to describe the specifics of the step. The
+contents of this field are included with the scheduled activity, and can be edited later.
+
+In the :guilabel:`Assignment` field, select one of the following options:
```

Comment:
The wording here comes across as a bit short. The instructions are clear but it's not clear why we're doing anything. Consider expanding a bit.

As well, we say "First select an **Activity Type**..." however this isn't really the first step in the instructions so that threw me off (we're saying "First do this" after 2-3 steps before it)